docs/sdk/authentication/auth-sig.md

```diff
@@ -60,13 +65,18 @@ const authSig = await ethConnect.signAndSaveAuthMessage({
   account: walletAddress,
   chainId: 1,
   expiration: new Date(Date.now() + 1000 * 60 * 60 * 24).toISOString(),
```

Comment:
This exp is now too high for the network to accept. we should limit this to ~ `30 days`

Reply:
It takes in milliseconds as its argument. So the exp is 24 hours from now